slr/basic_app/templates/basic_app/frontend_submission.html

```diff
@@ -1,11 +1,94 @@
+{% load static %}
 <!DOCTYPE html>
 <html lang="en">
     <head>
         <meta charset="UTF-8">
-        <title>Front End Submission</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>My Frontend Submission</title>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+        <style>
+            a:focus {
+                outline: none;
+            }
+            .nav-link {
+                font-size: 15px;
+            }
+            .navbar-nav > li{
+                margin-left: 15px;
+                margin-right: 15px;
+            }
+            .navbar {
+                padding-left: 10em;
+                background-color: #222;
+            }
+            .content {
+                margin-top: 7em;
+            }
+            .sections {
+                box-shadow: 0px,0px,0px,rgba(0,0,0,0.5);
+            }
+            .footer {
+                border-top: 1px solid grey;
+            }
+            @media (max-width: 600px) {
+                .navbar {
+                    padding-left: 2em;
+                }
+                .navbar-nav > li{
+                margin-left: 0;
+                margin-right: 0;
+                }
+            }
+        </style>
     </head>
     <body>
-        <h1>nothing to look at here.</h1>
-        <h2>At least not yet.</h2>
+        <nav class="navbar navbar-expand-md navbar-default navbar-dark">
+           <a href="{% url 'basic_app:index' %}" class="navbar-brand">WSoc</a>
+           <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#altnavbar">
+           <span class="navbar-toggler-icon"></span>
+           </button>
+           <div class="navbar-collapse collapse" id="altnavbar">
+                <ul class="navbar-nav">
+                    <li class="nav-item">
+                      <a class="nav-link" href="{% url 'basic_app:frontend' %}">Front End</a>
+                    </li>
+                    <li class="nav-item">
+                      <a class="nav-link" href="{% url 'basic_app:backend' %}">Back End</a>
+                    </li>
+                    <li class="nav-item">
+                      <a class="nav-link" href="{% url 'basic_app:frontend_submission' %}">Front End Submission</a>
+                    </li>    
+                  </ul>
+           </div>
+        </nav>
```

Comment:
Django allows for an extension of the base template.
This is a very powerful feature that doesn't require you to rewrite the same code in another file, and rather just implement it from a base template.
Look into this feature [here](https://tutorial.djangogirls.org/en/template_extending/).

This is done in this project as well!
You could redirect yourself to the back end template files and see that that's precisely the same concept that we've worked with!